scripts/build-windows-k8s.sh

```diff
@@ -66,8 +67,9 @@ k8s_16_cherry_pick() {
 	# 4f196c6cac Fix the issue that ping uses the incorrect NIC to resolve name sometimes
 	# 2c9fd27449 Workaround for Outbound Internet traffic in Azure Kubernetes
 	# 5fa0725025 Use adapter vEthernet (HNSTransparent) on Windows host network to find node IP
+	# 79cf9963f7 Merge pull request #51126 from chen-anders/anders/port-47991-to-release-1.6
 
-	git cherry-pick 5fa0725025..232fa6e5bc
+	git cherry-pick 79cf9963f7..232fa6e5bc
```

Comment:
Do you need to update k8s_17_cherry_pick too?

Reply:
No, v1.7 cherry-pick is working fine (we merged another PR that bumped Windows to v1.7.5 today).

Reply:
@seanknox Did we push the build branch for v1.7? I didn't seem to find it although I found v1.6. If not, could we push v1.7 too, just to make sure we have all the commits in.